test/hotspot/jtreg/runtime/linkResolver/InterfaceObjectTest.java

```diff
@@ -24,13 +24,13 @@
 /*
  * @test
  * @bug 8026394 8251414
- * @summary test interface resolution when clone and finalize are declared abstract within
+ * @summary test interface resolution when clone and toString are declared abstract within
  *          an interface and when they are not
  * @compile InterfaceObj.jasm
  * @run main InterfaceObjectTest
  */
 interface IClone extends Cloneable {
-    void finalize() throws Throwable;
+    String toString();
```

Comment:
One more minor comment.  This can be removed and the comment above referring to toString can also be removed.

Reply:
Done.